### ISSUE TITLE
[BITV] 9.3.3.1/8.1 - Modale dialog "Authentication required" - If the "Password" input field is left empty, an error message appears. This error message "Failed to authenticate, please try again" is unspecific and does not give any information about which input caused the error. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37080]
Active users - Users - Nextcloud
https://try.nextcloud.com/ltd/a11y/index.php/settings/users

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-c3b56e67ab-v8-n1

</details>

**Comment by JuliaKirschenheuter** [comment]
@michaelnissenbaum could you please check it if this issue is already resolved?

In my opinion it is but i want to be sure. Thanks a lot!

![Peek 2023-08-24 09-44](https://github.com/nextcloud/server/assets/6078378/ece7e4e8-2d54-431f-aeac-e21b95a6614f)

**Comment by JuliaKirschenheuter** [comment]
https://github.com/nextcloud/server/pull/38839

**Comment by michaelnissenbaum** [comment]
Hi @JuliaKirschenheuter. No, from my perspective, the issue still remains. This concerns the modal dialog with the error message "Failed to authenticate, please try again." The error message is unspecific and unclear in this case.
![CleanShot 2023-08-24 at 20 41 33@2x](https://github.com/nextcloud/server/assets/48948959/67f47d0a-de12-4aff-ad87-6d4c120e516d)

Additionally I found another issue. The button with the eye icon doesn't have alternative text - an empty "aria-label" attribute is used.

![CleanShot 2023-08-24 at 20 45 46@2x](https://github.com/nextcloud/server/assets/48948959/20641c3c-5681-4b16-94ae-3bfad6ed147f)

And the button with the arrow has an incorrect alternative text - aria-label="Clear text".
![CleanShot 2023-08-24 at 20 46 41@2x](https://github.com/nextcloud/server/assets/48948959/9e67db51-710d-4174-a110-0fa9c248d702)

**Comment by JuliaKirschenheuter** [comment]
Thank you @michaelnissenbaum!

**Comment by JuliaKirschenheuter** [comment]
@artonge  am i right that we would need much more backend differentiation for showing a detailed information? 
Or could i probably just change this message: 

![Screenshot from 2023-09-26 16-35-58](https://github.com/nextcloud/server/assets/6078378/fb0c56c6-d9d1-4953-81a4-9187a1f383d3)

to "your password was wrong. please try again"? Seems to be that we have only this one case.

**Comment by artonge** [comment]
I am not sure how this is an accessibility issue. There is only one field, so what's wrong is obvious in this case. No ?

In any case, we could do two things:

1. Prevent the submission of the form if the input is empty.
2. Clearly state that the password is wrong. No need for backend differentiation here, as only the password comes into play, so we know for sure that it is the password that is wrong. The current message comes from: https://github.com/nextcloud/server/blob/f134244c90ab4aff3fa7758f8c548ebd89c6a256/core/src/OC/password-confirmation.js#L126-L127
